### PR TITLE
support contacting multiple politicians

### PR DESCRIPTION
**Pull request by travisbloom-cb**
closes https://github.com/Stand-With-Crypto/swc-web/issues/642

## What changed? Why?
- The email action flow
- The call action flow
- The politician details page when you add your address

## How has it been tested?

- [X] Locally
- [ ] Vercel Preview Branch
- [ ] Unit test
- [ ] Functional test

## Change management

type=routine 
risk=low 
impact=sev5

### CODE REVIEW

File: src/actions/actionCreateUserActionEmailCongressperson.ts [hunk 1]
```diff
@@ -189,7 +187,7 @@ async function _actionCreateUserActionEmailCongressperson(input: Input) {
     opts: {
       isEmailOptin: true,
     },
-    emailSubject: 'Support Crypto', // This does not particularly matter for now as subject is currently overridden in the Capitol Canary admin settings.
+    emailSubject: 'Support FIT21',
```
Comment (travisbloom-cb) [inline src/actions/actionCreateUserActionEmailCongressperson.ts:190]: @benson-budiman-cb to change CC config once this is deployed

File: src/components/app/userActionFormCallCongressperson/constants.ts [hunk 2]
```diff
@@ -7,3 +9,5 @@ export enum SectionNames {
 
 export const ANALYTICS_NAME_USER_ACTION_FORM_CALL_CONGRESSPERSON =
   'User Action Form Call Congressperson'
+
+export const CALL_FLOW_POLITICIANS_CATEGORY: YourPoliticianCategory = 'senate'
```
Comment (travisbloom-cb) [inline src/components/app/userActionFormCallCongressperson/constants.ts:13]: TODO revert before merging, after people QA

File: src/utils/server/capitolCanary/campaigns.ts [hunk 3]
```diff
@@ -27,6 +27,8 @@ export const enum CapitolCanaryCampaignId {
 
   // Campaign IDS to use when sending emails to representatives.
   DEFAULT_EMAIL_REPRESENTATIVE = 137765,
+  DEFAULT_EMAIL_REPRESENTATIVE_AND_SENATORS = 145319,
+  DEFAULT_EMAIL_SENATORS = 145317,
 }
```
Comment (benson-budiman-cb) [inline src/utils/server/capitolCanary/campaigns.ts:32]: @travisbloom-cb I forgot to give you the sandbox campaign IDs as well - here they are:
- 145354 (REPRESENTATIVE_AND_SENATORS)
- 145353 (SENATORS)

File: src/actions/actionCreateUserActionEmailCongressperson.ts [hunk 4]
```diff
@@ -202,6 +200,21 @@ async function _actionCreateUserActionEmailCongressperson(input: Input) {
   return { user: getClientUser(user) }
 }
 
+function getCapitalCanaryCampaignId(politicianCategory: YourPoliticianCategory) {
+  if (NEXT_PUBLIC_ENVIRONMENT !== 'production') {
+    return SandboxCapitolCanaryCampaignId.DEFAULT_EMAIL_REPRESENTATIVE
+  }
+
+  switch (politicianCategory) {
+    case 'senate':
```
[the remark this answers is not in the document]
Comment (benson-budiman-cb) [reply src/actions/actionCreateUserActionEmailCongressperson.ts:209]: yeah this is correct. I also set you the sandbox campaign IDs if you want to include that for non-production environments.

when we QA this in testing/preview, the sandbox Capitol Canary Admin UI can actually tell us who the email would have been sent to, so we can take advantage of that when debugging.

File: src/components/app/userActionFormEmailCongressperson/constants.ts [hunk 5]
```diff
@@ -1,2 +1,6 @@
+import { YourPoliticianCategory } from '@/utils/shared/yourPoliticianCategory'
+
 export const ANALYTICS_NAME_USER_ACTION_FORM_EMAIL_CONGRESSPERSON =
   'User Action Form Email Congressperson'
+
+export const EMAIL_FLOW_POLITICIANS_CATEGORY: YourPoliticianCategory = 'senate'
```
[the remark this answers is not in the document]
Comment (benson-budiman-cb) [reply src/components/app/userActionFormEmailCongressperson/constants.ts:6]: Sending emails via Capitol Canary seems to work fine (see screenshot).

Similar question to the call flow - is the default email message supposed to be about FIT21, or about something else?

image

[the remark this answers is not in the document]
Comment (travisbloom-cb) [reply src/components/app/userActionFormEmailCongressperson/constants.ts:6]: currently we should still be sending to congressperson about fit

File: src/components/app/userActionFormCallCongressperson/constants.ts (hunk 2 above)
Comment (travisbloom-cb) [reply src/components/app/userActionFormCallCongressperson/constants.ts:13]: 1) this PR is just to get the logic in place so we can quickly pivot to new bills, so for now it's still FIT
2) yes, @coinbasemar aligned that this was to reduce confusion
3) https://github.com/Stand-With-Crypto/swc-web/pull/737/files#diff-cebc8926e7ecbda295b1147b77f13198f1dd8862b5a429f9120762aa6ab5af37R32

**Comment by benson-budiman-cb** [comment]
nit: oxford comma here

Screenshot 2024-04-03 at 10 31 27 AM

File: src/utils/shared/orderSenatorsByImportanceForOutreach.ts [hunk 6]
```diff
@@ -0,0 +1,73 @@
...
+  'sherrod---brown',
+  'jack---reed',
+  'mike---crapo',
+  'robert---menendez---1',
+  'mike---rounds',
+  'mark---warner',
+  'jon---tester',
+  'john---kennedy',
+  'bill---hagerty',
+  'elizabeth---wareen',
```
Comment (benson-budiman-cb) [inline src/utils/shared/orderSenatorsByImportanceForOutreach.ts:14]: @travisbloom-cb is this intentional or a typo? seems like this should be `elizabeth---warren`

Comment (travisbloom-cb) [reply src/utils/shared/orderSenatorsByImportanceForOutreach.ts]: Good catch, fixed

**Review by benson-budiman-cb — DISMISSED** [review]
😮

**Pull request merged**